testsuite/Cargo.toml

```diff
@@ -16,16 +16,24 @@ path = "tests/main.rs"
 harness = true
 
 [dependencies]
+anyhow = "1.0"
 assert_cmd = "2.0"
+dynosaur = "0.3"
```

Comment:
Wow, didn't know about that cool crate!

Reply:
It’s actually recommended over `async-trait` now! The implementer does not need to know about the crate too :slightly_smiling_face: